django/thunderstore/api/cyberstorm/views/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied, ValidationError
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.utils import conditional_swagger_auto_schema
+from thunderstore.social.views import DeleteAccountForm
+
+User = get_user_model()
+
+
+class CyberstormUserDeleteRequestSerialiazer(serializers.Serializer):
+    verification = serializers.CharField()
+
+
+class CyberstormUserDeleteResponseSerialiazer(serializers.Serializer):
+    username = serializers.CharField()
+
+
+class UserDeleteAPIView(APIView):
```

Comment:
You're missing permission classes (specifically at least the IsAuthenticated should be set to this view)

Reply:
Depending on if https://github.com/thunderstore-io/Thunderstore/pull/953 was merged, I'll either add it or not add it.

Reply:
All of the new views should now have `IsAuthenticated`